server/routes/matchRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const match = require('../controllers/match');
+const verifyToken = require('../middlewares/authentication');
+
+router.post('/newmatch', verifyToken, (req, res) => {
+  match.addMatch(req, res);
+});
```

Comment:
```suggestion
router.post('/newmatch', verifyToken, match.addMatch);
```
You can simplify it like this